### PR TITLE
update ls 1.16.18

### PR DESCRIPTION
**Pull request by saranshsaini**
make changes to server request args to enable use of 1.16.18
allows more models in chat (4o, claude)

### CODE REVIEW

**Review by fortenforge — APPROVED** [review]
<sub>_[Reviewed with Forge](https://codeium.com/forge/Exafunction/codeium.nvim/223/review)_</sub>

**Comment by fortenforge** [comment]
is it bad that this doesn't build

**Comment by pqn** [comment]
Yes pretty sure it should build

File: lua/codeium/source.lua [hunk 1]
```diff
@@ -84,7 +84,7 @@ local function buf_to_codeium(bufnr)
 		language = language,
 		text = text,
 		line_ending = line_ending,
-		absolute_path = vim.api.nvim_buf_get_name(bufnr),
+		absolute_uri = 'file://' .. vim.api.nvim_buf_get_name(bufnr),
```
Comment (pqn) [inline lua/codeium/source.lua:87]: Backslashes should be replaced by forward slashes for URI construction.

**Pull request merged**